sdk/resourcehealth/Azure.ResourceManager.ResourceHealth/samples/Generated/Samples/Sample_ResourceHealthEventResource.cs

```diff
@@ -80,7 +80,7 @@ public async Task GetSecurityAdvisoryImpactedResourcesBySubscriptionIdAndEventId
         public async Task FetchDetailsBySubscriptionIdAndTrackingId_EventDetailsBySubscriptionIdAndTrackingId()
         {
             // Generated from example definition: specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-10-01-preview/examples/Event_fetchDetailsBySubscriptionIdAndTrackingId.json
-            // this example is just showing the usage of "Event_FetchDetailsBySubscriptionIdAndTrackingId" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "Event_fetchDetailsBySubscriptionIdAndTrackingId" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `fetch`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/stable/2024-02-01/ResourceHealth.json#L683